### ISSUE TITLE
Plugin compatibility JDK 1.8 

### ISSUE THREAD

**Issue by maciek1839** [ascendtech/gwt-gradle#5]
Hello!
First of all, thank you for GWT Plugin! I got the following error :

```
\* What went wrong:
java.lang.UnsupportedClassVersionError: us/ascendtech/gwt/modern/GWTModernPlugin has been compiled by a more recent version of the Java Runtime (class file version 55.0), this version of the Java Runtime only recognizes
 class file versions up to 52.0
> us/ascendtech/gwt/modern/GWTModernPlugin has been compiled by a more recent version of the Java Runtime (class file version 55.0), this version of the Java Runtime only recognizes class file versions up to 52.0
```

To my knowleadge, the plugin requires Java 11 but I use Java 8.

Am I right? Do you know how to make it work with Java  8?

Best regards,
Maciej

**Comment by payammeyer** [comment]
@MaciekKuna it's definitely supposed to be Java 8 compatible, are you using the most recent release?Also, it looks like some of our other libraries that you can just pull in using the plugin are not compatible, for example AG-Grid. Do you happen to be bringing in anything else? 

For what it's worth, I was able to build a simple project without any add-ons using JDK 8.

**Comment by payammeyer** [comment]
Recompiled all other libraries and released version 0.4.17 with full JDK8 support. 

@MaciekKuna please take a look and let us know if you run into any issues.

**Comment by maciek1839** [comment]
It's working fine. Thank you! Although I have some questions about configuration but I will create a new issue.
